sources/pmwebapi.go

```diff
@@ -137,6 +137,17 @@ func newPcpSource() (PcpSource, error) {
 			log.Fatal("NewRequest: ", err)
 		}
 
+		unit := ""
+		if pcpMetrics.Units != "" {
+			unit = "_" + pcpMetrics.Units
```

Comment:
We'll want to take a look here, I don't think *everything* will have a meaningful unit. This is a place where unit tests may be able to help you faster than manual testing.